src/Form/selectors-css.js

```diff
@@ -6,7 +6,6 @@ const SUBMIT_BUTTON_SELECTOR = `
 input[type=submit],
 input[type=button],
 button:not([role=switch]):not([role=link]),
-[class*=submit i][class*=button i],
```

Comment:
This was matching a bunch of false positives for button wrappers and the likes. It has a positive effect on the test cases so it's a net win.